sar_pipeline/aws/metadata/stac.py

```diff
@@ -497,6 +491,31 @@ def add_collection_link(self, target="./collection.json"):
             )
         )
 
+    def rename_backscatter_assets(self, burst_folder: Path):
+        """Rename the backscatter assets in the burst folder to include the calibration
+        type in the filename. e.g. HH.tif -> HH_gamma0.tif
+
+        Parameters
+        ----------
+        burst_folder : Path
+            path to the local folder containing output products for a single burst.
+            e.g. /path/to/my/scene_burst/t070_149813_iw2
+        """
+
+        # get the list of files
+        burst_files = [x for x in burst_folder.iterdir()]
+        # get gamma0 or sigma0 calibration
+        calibration = self.h5.search_value("outputBackscatterNormalizationConvention")
```

Comment:
should we call this parameter "normalization_convention" to be explicit? I think calibration might not be the right terminology.